src/openclsim/plot/vessel_planning.py

```diff
@@ -41,21 +42,38 @@ def get_segments(df, activity, y_val):
 
 
 def get_gantt_chart(
-    vessels,
+    concepts,
     activities=None,
     id_map=None,
     colors=None,
     web=False,
     static=False,
     y_scale="text",
 ):
-    """Create a plot of the planning of vessels."""
-    id_map = id_map if id_map else {}
-    act_map = {ves.id: ves.name for ves in vessels}
+    """Create a plotly GANTT chart of the planning of vessels.
+    
+    Parameters
+    ----------
+    concepts
+        a list of vessels, sites or activities for which to plot all activities
+    activities
+        additional activities to be plotted, if not yet in concepts
+    id_map
+        by default only activity in concepts are resolved. Activities
+        associated with vessels and sites are not resolved. id_map solves this:
+        * a list of top-activities of which also all sub-activities 
+          will be resolved, e.g.: [while_activity]
+        * a manual id_map to resolve uuids to labels, e.g. {'uuid1':'name1'}
+    """
+    if type(id_map)==list:
+        id_map = {act.id: act.name for act in get_subprocesses(id_map)}
+    else:
+        id_map = id_map if id_map else {}
+    act_map = {ves.id: ves.name for ves in concepts}
 
     if activities is None:
         activities = []
-        for obj in vessels:
+        for obj in concepts:
```

Comment:
What are `concepts`?

Reply:
get_gantt_chart() can handle vessels, sites and activities. How do we name those in 1 word?